components/script/dom/bindings/trace.rs

```diff
@@ -160,12 +160,6 @@ impl<T: JSTraceable> JSTraceable for Rc<T> {
     }
 }
 
-impl JSTraceable for JobQueue {
-    fn trace(&self, trc: *mut JSTracer) {
-        self.0.trace(trc)
-    }
-}
-
 impl JSTraceable for Job {
```

Comment:
Yes, i answered in the previous one that i can auto derive for `JobQueue`, but not for `Job`. Adding derive attribute on `Job` still says, of `no method called trace found .. ...` . So i have to manually derive for `Job`.